app/controllers/__init__.py

```
@@ -1,5 +1,6 @@
 from .upload_files import *
```

Ruff (0.13.1)

`from .upload_files import *` used; unable to detect undefined names

(F403)